### ISSUE TITLE
Looks like ARM aarch64 is not supported

### ISSUE THREAD

**Issue by clst** [oracle/oci-grafana-metrics#110]
```
lvl=eror msg="Could not start plugin" logger=plugin.manager pluginId=oci-metrics-datasource err="fork/exec /var/lib/grafana/plugins/oci-metrics-datasource/oci-plugin_linux_arm64: no such file or directory"
```

Does that mean the plugin does not work on Ampere A1 instances?

**Comment by EdsonBittencourt** [comment]
Same issue here.

**Comment by EdsonBittencourt** [comment]
@mayurOciSA Thanks for the update. Do you know if there's plans to make it work?

**Comment by BetaAthe** [comment]
I have the same issue. Grafana can't find the oci-plugin_linux_arm64 file at boot.

**Comment by nce** [comment]
Same issue for me, thats a bit unfortunate :(

**Comment by EdsonBittencourt** [comment]
Datasource for logs is working after the last update.

**Comment by BetaAthe** [comment]
Not for me. Public release still doesn't have the arm binary.

**Comment by huskerdhf** [comment]
ARM support for the metrics plugin will be available in the next release.

**Comment by mamorett** [comment]
ARM support is now available and released, in version 3.0.6 of the plugin.
